cli/src/commands/next.rs

```diff
@@ -60,6 +62,42 @@ pub(crate) struct NextArgs {
     edit: bool,
 }
 
+pub fn choose_commit<'a>(
+    ui: &mut Ui,
+    cmd: &str,
+    commits: &'a [Commit],
+) -> Result<&'a Commit, CommandError> {
+    writeln!(ui.stdout(), "ambiguous {cmd} commit, choose one to target:")?;
+    for (i, commit) in commits.iter().enumerate() {
+        writeln!(
+            ui.stdout(),
+            "{}: [{}] {}",
+            i + 1,
+            short_commit_hash(commit.id()),
+            commit.description().lines().next().unwrap_or_default()
```

Comment:
perhaps, `write_commit_summary()` can be used here

Reply:
Done.